templates/Procfile

```diff
@@ -1,2 +1,2 @@
-web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
+web: bundle exec puma -C ./config/puma.rb
```

Comment:
Without the port here, `-p $PORT`, or set in puma.rb things go wrong pretty quick up on Heroku. I would have guessed Heroku would have assigned it as a default somewhere but that doesn't look to be the case. I deployed without `-p $PORT` and with, and it would only work *with*.